cve_bin_tool/version.py

```diff
@@ -2,10 +2,10 @@
 import textwrap
 from urllib import request
 from packaging import version
-
+import fnmatch
 from cve_bin_tool.log import LOGGER
 
-VERSION = "1.1"
+VERSION = "1.1-dev"
```

Comment:
We might need to update this though.

Reply:
Yeah, I think probably 2.0alpha is more accurate at this point.  Although I don't *think* we're expecting any db changes so 2.0 might also be fine.

Reply:
I'm going to update our version to 2.0a today, so this is going to give you a merge conflict.

Incidentally, since I was looking up version conventions, I've learned that dev versions are covered by pep 440, so when we do switch back to a dev version it will be 2.0.dev0
https://www.python.org/dev/peps/pep-0440/#developmental-releases

The plan will be to switch the main tree to 2.0.dev0 sometime after I push a 2.0 release and make the 2.0 branch, i think.